packages/storybook/stories/inbox/full-page-inbox.stories.tsx

```diff
@@ -36,9 +36,9 @@ export const Hooks = () => {
   useEffect(() => {
     const intervalId = setInterval(() => {
       setAuth(
-        "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzY29wZSI6InVzZXJfaWQ6NzBmNmE0ZjQtMjkwNy00NTE4LWI4ZjMtYjljZmFiMjI0NzY0IGluYm94OnJlYWQ6bWVzc2FnZXMiLCJ0ZW5hbnRfc2NvcGUiOiJwdWJsaXNoZWQvcHJvZHVjdGlvbiIsInRlbmFudF9pZCI6Ijc2ODI1MWNmLTNlYjgtNDI2YS05MmViLWZhYTBlNzY3ODc2OCIsImlhdCI6MTY3MjI1NzY1OSwianRpIjoiYmJlMDMyMmMtZWY4Mi00M2FkLWI3NGMtOGZlYWNiNTczYTY0In0.Xs_yd8IhdNORK8LyleS10FDLQbb4sXkCtGHPq7tUGa4"
+        "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzY29wZSI6InVzZXJfaWQ6NzBmNmE0ZjQtMjkwNy00NTE4LWI4ZjMtYjljZmFiMjI0NzY0IGluYm94OnJlYWQ6bWVzc2FnZXMiLCJ0ZW5hbnRfc2NvcGUiOiJwdWJsaXNoZWQvcHJvZHVjdGlvbiIsInRlbmFudF9pZCI6Ijc2ODI1MWNmLTNlYjgtNDI2YS05MmViLWZhYTBlNzY3ODc2OCIsImlhdCI6MTY3MzY1NjQwNywianRpIjoiN2IxNTZhMTEtOGMwYS00MTA0LWI3OWEtYmVmMTNhM2MzMzNlIn0.cf46AojLX2fVDZ08B9763HiQI-Rzxr-Douwd4PC52BY"
       );
-    }, 300000 * 3); // 15 minutes
+    }, 180000); // 15 minutes
```

Comment:
Is the comment still accurate?

Reply:
its just in my dev env i was using to test. ill remove the comment